tests/integration/defs/perf/test_visual_gen_perf_sanity.py

```
@@ -0,0 +1,593 @@
+
+
+@pytest.mark.parametrize("visual_gen_perf_sanity_test_case", VISUAL_GEN_PERF_SANITY_TEST_CASES)
+def test_visual_gen_e2e(output_dir, visual_gen_perf_sanity_test_case):
+    """Run one VisualGen perf sanity case end-to-end."""
+    config = VisualGenPerfSanityTestConfig(visual_gen_perf_sanity_test_case, output_dir)
+
+    try:
+        config.parse_config_file()
+        config.run()
+        config.upload_test_results_to_database()
+    except Exception as exc:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)